bookwyrm/librarything_import.py

```diff
@@ -0,0 +1,39 @@
+''' handle reading a csv from librarything '''
+import csv
+import re
+import math
+
+from bookwyrm import models
+from bookwyrm.models import ImportItem
+from bookwyrm.importer import Importer
+
+
+class LibrarythingImporter(Importer):
+    service = 'LibraryThing'
+    delimiter = '\t'
+    encoding = 'ISO-8859-1'
+    # mandatory_fields : fields matching the book ISBN13, title and author
+    mandatory_fields = ['ISBN', 'Title', 'Primary Author']
+
+    def parse_fields(self, initial):
+        data = {}
+        data['import_source']   = self.service
+        data['Book Id'] = initial['Book Id']
+        data['Title'] = initial['Title']
+        data['Author'] = initial['Primary Author']
+        data['ISBN13'] = initial['ISBN']
+        data['My Review'] = initial['Review']
+        data['My Rating'] = math.ceil(float(initial['Rating']))
```

Comment:
This will throw an error when no rating is set for a book:
```
ValueError at /import
could not convert string to float: ''
```

Reply:
fixed